playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -109,7 +109,7 @@
   gather_facts: no
   tasks:
   - set_fact:
-      openshift_rpm: "{{ hostvars[groups['nodes'][0]]['yum'].results | selectattr('name', 'match', '^(origin|atomic-openshift)$') | first }}"
+      openshift_rpm: "{{ hostvars[groups['nodes'][0]]['yum'].results | selectattr('name', 'match', '^(origin-hyperkube|atomic-openshift-hyperkube)$') | first }}"
```

Comment:
`hyperkube` contains only the kubernetes components, right?

Reply:
``` 
1. openshift/origin and openshift/node are now dead (in both OCP and Origin)
2. openshift/origin's closest equivalent is openshift/origin-control-plane which has the openshift and oc binaries in it
3. openshift/node is now called openshift/origin-node, and has some other changes described below
4. the hypershift binary (launches the openshift api and controllers) is in openshift/origin-hypershift
5. the hyperkube binary (launches the kube api, controllers, and kubelet) is in openshift/origin-hyperkube
6. two new images openshift/origin-test and openshift/origin-cli have been created that contain the e2e binary and the cli binary
```
So origin RPM is no more. should we better move to generic `origin` and `openshift` name or `hypershift/hyperkube`?

Reply:
Ah, I think this is correct now. We should just need the kubelet binary and all the rest of the components run in container images.

Reply:
Hrm, we probably also need `oc` in there.

Reply:
ps. hypershift RPM does not exist in 3.11. Not sure if it is expected or does not exist YET.